rest/grpc-rest-bindings.yml

```diff
@@ -0,0 +1,166 @@
+---
+type: google.api.Service
+config_version: 3
+
+name: api.v1.trading
+title: Vega Trading API
+
+apis:
+  - name: TradingService
+  - name: TradingDataService
+
+http:
+  rules:
+    # Markets
+    - selector: api.v1.TradingDataService.Markets
+      get: '/markets'
+    - selector: api.v1.TradingDataService.MarketByID
+      get: '/markets/{market_id}'
+    - selector: api.v1.TradingDataService.Candles
+      get: '/markets/{market_id}/candles'
+    - selector: api.v1.TradingDataService.MarketDepth
+      get: '/markets/{market_id}/depth'
+    - selector: api.v1.TradingDataService.OrdersByMarket
+      get: '/markets/{market_id}/orders'
+    - selector: api.v1.TradingDataService.OrderByMarketAndID
+      get: '/markets/{market_id}/orders/{order_id}'
+    - selector: api.v1.TradingDataService.TradesByMarket
+      get: '/markets/{market_id}/trades'
+    - selector: api.v1.TradingDataService.LastTrade
+      get: '/markets/{market_id}/trades/latest'
+    - selector: api.v1.TradingDataService.MarketAccounts
+      get: '/markets/{market_id}/accounts'
+
+    # assets
+    - selector: api.v1.TradingDataService.Assets
+      get: '/assets'
+    - selector: api.v1.TradingDataService.AssetByID
+      get: '/assets/{id}'
+    - selector: api.v1.TradingDataService.FeeInfrastructureAccounts
+      get: '/assets/{asset}/infrastructureFeeAccount'
```

Comment:
Is the `{asset}` correct in here?

Reply:
Yes, it's correct.

- `AssetByIDRequest` has `string id`: https://github.com/vegaprotocol/vega/blob/e5bc06543b7b1ae87d4fb4888168d04586bd7b6f/proto/api/trading.proto#L380-L384
- `FeeInfrastructureAccounts` has `string asset`: https://github.com/vegaprotocol/vega/blob/e5bc06543b7b1ae87d4fb4888168d04586bd7b6f/proto/api/trading.proto#L984-L990